open-metadata-implementation/admin-services/Egeria-admin-services-server-configuration.postman_collection.json

```diff
@@ -1952,7 +1952,7 @@
 								],
 								"body": {
 									"mode": "raw",
-									"raw": "      { \n         \"class\":\"ViewServiceConfig\",\n         \"omagserverName\":\"cocoMDS1\",\n         \"omagserverPlatformRootURL\":\"https://localhost:9443\"\n      }"
+									"raw": "      { \n         \"class\":\"ViewServiceConfig\",\n         \"omagserverName\":\"myMetadataAccessPoint\",\n         \"omagserverPlatformRootURL\":\"myPlatformRootURL1\"\n      }"
```

Comment:
This should be using postman variables from your environment definition ie  {{platformURL}} {{serverName}} etc
Use the variables defined in the current env files & extend with any new concepts like view servers etc - that way someone just needs to update their environment definition, not edit the actual collection?